src/lemlib/odom/differentialArc.cpp

```diff
@@ -174,4 +174,13 @@ void DifferentialArc::update() {
     // calculate global position
     pose += local.rotate(avgTheta);
 }
+
+int DifferentialArc::getPollRate() const {
+    int maxPollRate = 0;
+    for (auto sensor : this->horizontals) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
+    for (auto sensor : this->verticals) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
+    for (auto sensor : this->gyros) maxPollRate = std::max(maxPollRate, sensor->getPollRate());
+    for (auto sensor : this->drivetrain) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
```

Comment:
```suggestion
    for (auto& sensor : this->horizontals) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
    for (auto& sensor : this->verticals) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
    for (auto& sensor : this->gyros) maxPollRate = std::max(maxPollRate, sensor->getPollRate());
    for (auto& sensor : this->drivetrain) maxPollRate = std::max(maxPollRate, sensor.getPollRate());
```
I may be wrong here, but shouldn't we copy by reference here?

Reply:
Don't know, but it doesn't throw me a compile error